index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="./styles.css">
+  <title>Portfolio</title>
+  
+</head>
+<main>
+  <nav>
+  <p class="my-logo"><a href="#">Alexander</a></p>
+<div class="hamburger-menu">
+  <hr>
+  <hr>
+  <hr>
+</div>
+  </nav>
+  <header>
+    <h1> Alexander Odufuye </h1>
+   <p class="head-text">Software Developer</p>
+  </header>
+<section class="intro">
+  <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me..</p>
+</section>
+<hr class="border">
+<section class="items">
+<div class="list-languages">
+  <ul>
+    <li class="mini-tag">Language</li>
+    <li>Javascript</li>
+    <li>Ruby</li>
+    <li>Html</li>
+    <li>Css</li>
+  </ul>
+</div>
+<div class="list-frameworks">
+  <ul>
+    <li class="mini-tag">Frameworks</li>
+    <li>Bootstrap</li>
+    <li>Ruby on Rails</li>
+    <li>RSPec</li>
+    <li>CapyBara</li>
+    <li></li>
+  </ul>
+</div>
+<div class="list-skills">
+  <ul>
+    <li class="mini-tag">Gitlabs</li>
+    <li>Codekit</li>
+    <li>Github</li>
+    <li>Codepen</li>
+    <li>Terminal</li>
+  </ul>
+</div>
+</section>
+<hr class="border">
+<section class="contact-us">
+  <div class="icon"> 
+    <img src="./images/github.png" alt="...">
+   <img src="./images/Vector.png" alt="image">
+   <img src="./images/Vecr.png" alt="image">
+  </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)